environment.rb

```diff
@@ -0,0 +1,98 @@
+require 'securerandom' #Gem to generate ids
+
+module Environment
+  class Depot
+      attr_accessor :packs
```

Comment:
Indentation should be two spaces, be careful when setting in your IDE the tad attribute, sometimes Git does recognize any tab as a fours-spaced one.